dev_apps/music-nga/js/app.js

```diff
@@ -112,6 +163,10 @@ function setHeaderTitle(title) {
   window.requestAnimationFrame(() => headerTitle.textContent = title);
 }
 
+function setBackButtonHidden(hidden) {
+  header.els.actionButton.style.visibility = hidden ? 'hidden' : 'visible';
+}
```

Comment:
Rather than making a function here, we could add a `.hidden` CSS class and then just use `node.classList.toggle('hidden', <expr>)`.